build.gradle

```diff
@@ -49,9 +56,9 @@ dependencies {
 
   // Add dependencies here
   implementation 'com.google.maps:fleetengine-auth:1.3.3'
-  implementation project(':gapic-google-maps-fleetengine-v1-java')
-  implementation project(':grpc-google-maps-fleetengine-v1-java')
-  implementation project(':proto-google-maps-fleetengine-v1-java')
+
+  implementation 'com.google.api:gax:1.55.0'
```

Comment:
is it expected that we need to include this dep too?

Reply:
@bshi

Reply:
This is a known issue (encountered by @shawnlu-g IIRC).  Fine to have this workaround IMO.